sot/opcode_translator/executor/variables/basic.py

```diff
@@ -95,6 +108,14 @@ def wrap_literal(value: Any) -> ConstantVariable:
         return ConstantVariable(value, ConstTracker(value))
 
 
+implemented_property = set()
```

Comment:
```suggestion
IMPLEMENTED_TENSOR_PROPERTIES = set()
```

这个文件除去 TensorVariable 还有别的 Variable，改一下变量名比较好